portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,100 @@
+            Cal State Long Beach. Currently, I am <br> participating in a progam by Google called SPS and I am on team 41! </p>
+
+      <hr>
+      <h3 class="titles">Hobbies!</h3>
+      <p class="hobbies">
+          <b class="subtitles">Sports</b>
+          <br > I really enjoy playing any sport because of how competative I am. I did <br> play Soccer for 9 
+          years and Volleyball for my highschool senior season.</br>
+         <b class="subtitles">Baking<img src="/decor/bluebutterflies.png" width="100" height="125" style="float:right"></b>
+         <br>For sure something I am trying to get better at but I love being able to bake <br>and I really want to start getting into making food in general:)</br>
+         <b class="subtitles">DIY</b>
+         <br>I love doing projects whether its something I need or something just for<br> fun, I find it really enjoyable</br></p>
+      <hr>
+      <h3 class="titles">Music!</h3>    
+        <div class="container">
+            <div class="artist">Lauryn Hill</div>
+            <div class="artist">Kendrick Lamar</div>
+            <div class="artist">Kid Cudi</div>
+            <div class="artist">J.cole</div>
+            <div class="artist">Camila</div>
+            <div class="artist">Julieta Venegas</div>
+            <div class="artist">N.W.A</div>
+        </div>    
+      <hr>
+      <h3 class="titles">Contact Me !</h3>
+        <div class="contactname">
+          <div class="social"> LinkedIn:</div>
+          <div class="social"> GitHub:</div>
+          <div class="social"> Instagram:</div>
+         </div>
+         <div class="links">
+          <div class="link"><a href="https://www.linkedin.com/in/carine-gordillo-b465361b8/"> Carine Gordillo</a></div>
+          <div class="link"><a href="https://github.com/carinegordillo"> carinegordillo</a></div>
+          <div class="link"><a href="https://www.instagram.com/carinegordillo"/> @carinegordillo</a></div>
+        </div>         
+      <hr>
+      <h3 class="titles">Projects!</h3>
+      <b class="subtitles">Simple Login System</b>
+      <br>- simple code that prompts user for their username and password or to <br>create a new account<br>
+      -uses separate text files to store data of users</br>
+      <b class="subtitles">Alien Invaders</b>
+      <br>-Game based off of the classic Space Invaders made by using the <br>interpreter Pygame</br>
+      <b class="subtitles">Dice Game</b>
+      <br>-A gambling dice game by using the random function, allows<br> user to bet against a computer and only allows bets<br>that both players can afford</br>
+      <hr>
+      <h3 class="titles">Gallery!</h3>
+
+    <div class="row"> 
+         <div class="column">
+             <img src="/images/cat.jpg" style="width:30%">
+             <img src="/images/Mamba.JPG" style="width:25%">
```

Comment:
This guy being 25% while the others are 30% is a little sus - if it looks good on your site, then please ignore this comment! Otherwise, try it at 30% and hopefully all these can be the same